src/ResourceManager/Cdn/Commands.Cdn/Endpoint/ConfirmAzureRmCdnEndpointProbeUrl.cs

```diff
@@ -0,0 +1,40 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Cdn.Common;
+using Microsoft.Azure.Commands.Cdn.Helpers;
+using Microsoft.Azure.Commands.Cdn.Models.Endpoint;
+using Microsoft.Azure.Commands.Cdn.Properties;
+using Microsoft.Azure.Management.Cdn;
+using Microsoft.Azure.Management.Cdn.Models;
+
+namespace Microsoft.Azure.Commands.Cdn.Endpoint
+{
+    [Cmdlet(VerbsLifecycle.Confirm, "AzureRmCdnEndpointProbeURL"), OutputType(typeof(PSValidateProbeOutput))]
```

Comment:
@prakharsharma10 what does this cmdlet do? If we are checking the existence of a resource or object, then the verb for this cmdlet should be `Test` instead of `Confirm`

Reply:
This command validates the probe URL. We have the implementations in the REST API and SDKs which use the name "validate". Have used a substitute for that as listed here https://docs.microsoft.com/en-us/dotnet/api/system.management.automation.verbslifecycle.confirm?redirectedfrom=MSDN&view=powershellsdk-1.1.0